instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_aio_client.py

```diff
@@ -55,20 +54,19 @@ def callback(call):
 
 class _BaseAioClientInterceptor(OpenTelemetryClientInterceptor):
     @staticmethod
-    def propagate_trace_in_details(client_call_details):
+    def propagate_trace_in_details(client_call_details: ClientCallDetails):
         metadata = client_call_details.metadata
         if not metadata:
-            mutable_metadata = OrderedDict()
+            mutable_metadata = Metadata()
         else:
-            mutable_metadata = OrderedDict(metadata)
+            mutable_metadata = Metadata(*tuple(metadata))
```

Comment:
It looks like this was based on the previous assumption that `metadata` was an immutable tuple. `Metadata` looks to be mutable and retains previously set values (as a multimap)–can we just pass the metadata directly?

Reply:
I guess the current approach would work with tuple already. I'm OK to just leave it as is.